chat-ui/tsconfig.node.json

```diff
@@ -6,5 +6,5 @@
     "moduleResolution": "bundler",
     "allowSyntheticDefaultImports": true
   },
-  "include": ["vite.config.ts"]
+  "include": ["vite.config.component.ts"]
```

Comment:
What this rename?

Reply:
b/c now there are 2 vite configs: 1 for the webapp, 1 for component. renamed the configs to make clear which for which